### DISCUSSION TITLE
N to 1 asset preprocessing.

### DISCUSSION THREAD

**Discussion by brandon-reinhart** [bevyengine/bevy#12341] [General]
Is there a way, currently, to create an N to 1 relationship between asset source files and a processed asset? This could be something like an audio file and a cue sheet, multiple textures and an atlas, or meta images and the resulting meta data.

The goal is that when one of the sources changes, the preprocessing watcher determines that the asset needs to be reprocessed.

All of this is in the context of a preprocessing asset server, not a standard runtime asset server.

**Comment by IronGremlin** [comment]
FWIW this is also a question that I have had and been unable to answer.
